Python/Source/Box.cpp

```diff
@@ -1,7 +1,11 @@
 #include "Box.hpp"
+#include "Rover/Constant.hpp"
 
 using namespace pybind11;
 using namespace Rover;
 
 void Rover::export_box(module& module) {
+  pybind11::class_<Box<pybind11::object>>(module, "Box")
+    .def(pybind11::init<Constant<pybind11::object>>())
```

Comment:
Remove the constructor for ```Constant<object>```.

Reply:
But then, when I try to create a Constant in Python as
c = rover.Constant(5)
I get: "rover.Box: No constructor defined!"
Is that OK too?

Reply:
Yep, that will be handled later by creating conversions.